src/test/java/org/tarantool/jdbc/JdbcResultSetMetaDataIT.java

```diff
@@ -201,6 +200,7 @@ public void testCaseSensitiveColumns() throws SQLException {
     @Test
     @DisplayName("returned case insensitive columns")
     public void testCaseInsensitiveColumns() throws SQLException {
+        assumeServerVersionLessThan(testHelper.getInstanceVersion(), ServerVersion.V_2_2);
```

Comment:
It looks that it is better to create a table with DOUBLE / NUMBER column instead of disabling the entire test case on tarantool-2.2+. The same for the test cases below.

Reply:
> Support 'number' on 2.2.1 and above. Disable float/real/double on 2.2.0 and above.

so maybe 2.2.1+?

Reply:
We cannot say anything about 2.2.0.x — because here we enable NUMBER and disable FLOAT/REAL/DOUBLE. One of them is supported, but what exacly? So <2.2.0 — DOUBLE, >=2.2.1 — NUMBER.